.github/workflows/generate_files.yml

```diff
@@ -0,0 +1,51 @@
+name: Build Chain Generate Files
+
+on:
+  pull_request:
+    paths:
+      - '.ci/**'
+
+jobs:
+  build-chain:
+    runs-on: ubuntu-latest
+    name: File Generation
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          ref: ${{ github.event.pull_request.head.ref }}
+      - name: Image Generation
+        uses: kiegroup/build-chain-files-generator@main
+        with:
+          definition-file: https://raw.githubusercontent.com/kiegroup/droolsjbpm-build-bootstrap/master/.ci/pull-request-config.yaml
+          file-type: image
+          output-file-path: ./docs/project-dependencies-hierarchy.png
+      - name: Repository List Generation
+        uses: kiegroup/build-chain-files-generator@main
+        with:
+          definition-file: https://raw.githubusercontent.com/kiegroup/droolsjbpm-build-bootstrap/master/.ci/pull-request-config.yaml
+          file-type: repository-list
+          output-file-path: ./script/repository-list.txt
+          exclude: "@master:7.x"
+      - name: Repository List Generation
```

Comment:
could we differentiate as 'Optaplanner Repository List Generation' ?

Reply:
done, good catch!